server/operator.go

```diff
@@ -225,6 +226,7 @@ type regionOperator struct {
 	Ops    []Operator    `json:"ops"`
 	Kind   ResourceKind  `json:"kind"`
 	State  OperatorState `json:"state"`
+	mu     *sync.RWMutex
```

Comment:
Umm. maybe for print more friendly.
the operator String() function  use `fmt.Sprintf`, and golint report 
```
call of fmt.Sprintf copies lock value: server.regionOperator contains sync.RWMutex.
```